frontend/src/v5/ui/routes/viewer/tickets/ticketsForm/properties/manyOfProperty.component.tsx

```diff
@@ -30,9 +31,10 @@ export const ManyOfProperty = ({ values, ...props }: ManyOfPropertyProps) => {
 	} else {
 		items = (values as string[]).map((value) => ({ value, label: value }));
 	}
+	const filterImpossibleValues = intersection(props.value, items.map((i) => i.value));
```

Comment:
Can we change this to selectValues or validValues? 'filterImpossibleValues' has a name that seem more suitable for a function.

Also add a small comment that old values could be different because of changes in members of the teamspace or the template.